pom.xml

```diff
@@ -49,7 +49,9 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
```

Comment:
I don't think it's such a good idea. Especially since it does not solve our issue.

We'll be lucky if we can get an updated asciidoctor-ant version with latest 1.5.5. I wouldn't bet on having a version with the beta.

Let's extract this change in another PR and keep it on hold for now.